.github/workflows/ci_tests.yml

```diff
@@ -135,7 +144,8 @@ jobs:
       run: |
         python -m pip install -U pip
         pip install -U setuptools wheel
-        pip install -e .[dev]
```

Comment:
why are 'requirements.*' preferred over setup.py?

Reply:
Copy of my answer to Francesca:

I splittet it into two lines, because I wanted to install the requirements with the update (-U) option, which doesn't make sense for development (-e .) installation.

But it might not be necessary. I tried to change a lot of things before the CI on GitHub finally went through. It could very well be the change in line 91-92 that did the work...